src/ml_board/frontend/Dashboard/src/components/modelCard/ModelCard.tsx

```diff
@@ -85,6 +102,7 @@ export default function ModelCard() {
                 setTrainingDetails(resp_data.training_details);
                 setEvalDetails(resp_data.eval_details);
                 setDatasetDetails(resp_data.dataset_details);
+                setPipelineDetails(resp_data.pipeline_details);
                 setSysInfoCarbonFootPrintDetails(resp_data.experiment_environment.carbon_footprint);
                 setSysInfoEntryPointCmdDetails(resp_data.experiment_environment.entry_point_cmd);
                 Object.keys(resp_data.experiment_environment.system_env).map((sysInfoKeyName) => {
```

Comment:
`map` returns an array, but you are not using it, maybe you mean `forEach` but I'd recommend a normal `for` loop better